.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "lib/ds-proxy"]
+	path = lib/ds-proxy
+	url = https://github.com/CirclesUBI/ds-proxy.git
```

Comment:
Are there any changes here that cannot be upstreamed?

Reply:
Yeah, the relative paths..